lib/rules/selector-anb-no-unmatchable/__tests__/index.js

```diff
@@ -40,6 +40,14 @@ testRule({
 		{
 			code: 'a:nth-child(even) {}',
 		},
+		{
+			code: 'a:nth-child(3n + - 6) {}',
+			description: 'CSSTree parse error not flagged',
+		},
+		{
+			code: '.mixin(@variable: x):nth-child(0) {}',
+			description: 'nonstandard syntax (LESS parametric mixin)',
+		},
```

Comment:
```suggestion
```

Let's avoid adding CSS-like code in our tests.

Reply:
Ah! I think I misinterpreted your comment earlier, thanks for pointing this out. Have removed it.